src/routes/profile.ts

```diff
@@ -60,7 +64,8 @@ const router = Router();
  *       400:
  *         description: Bad request
  */
-router.post("/", checkJwt, registerProfile);
+//router.post("/", checkJwt, registerProfile);
+router.post("/", upload.any(), registerProfile);
```

Comment:
why any? isn't an array? and multer is a middleware, should be moved to the middleware folder -  look how Vika did it in the project

Reply:
+done